modals.js

```diff
@@ -63,25 +63,24 @@ const helpModal = (screen, blessed, prog) => {
   });
   helpLayout.focus();
   helpLayout.key(['escape'], () => {
-    prog.setModalOpen(false);
-    prog.returnFocus();
+    application.setIsModalOpen(false);
+    application.returnFocus();
     helpLayout.destroy();
   });
 };
 
-const eventTemplate = (busName) => `
-{
-  Entries: [
+const eventTemplate = (busName) => `{
+  "Entries": [
     {
-      "EventBusName": ${busName.substring(2)},
+      "EventBusName": "${busName}",
       "DetailType": "Scheduled Event",
-      "Source": "aws.events",
-      "Detail": {}
-    },
+      "Source": "custom.hello",
```

Comment:
Can we store what is here as a variable and display the same thing when they open again,